satsense/features/lacunarity.py

```diff
@@ -73,6 +74,7 @@ def lacunarities(canny_edge_image, box_sizes):
 
 class Lacunarity(Feature):
     """Lacunarity feature."""
+
```

Comment:
Remove whitespace here

Reply:
No, there should be a blank line after a class docstring according to PEP257 which this repo is following